deeprank/generate/DataGenerator.py

```diff
@@ -69,17 +65,16 @@ def __init__(self, chain1, chain2,
 
             >>> from deeprank.generate import *
             >>> # sources to assemble the data base
-            >>> pdb_source     = ['./1AK4/decoys/']
-            >>> pdb_native     = ['./1AK4/native/']
-            >>> pssm_source    = ['./1AK4/pssm_new/']
+            >>> mutant = PdbMutantSelection(pdb_path="1AK4.pdb",
+            >>>                             chain_id="C",
+            >>>                             residue_number=10,
+            >>>                             mutant_amino_acid="T",
+            >>>                             pssm_paths_by_chain={"C": "pssm_new/1AK4.C.pssm",
+            >>>                                                  "D": "pssm_new/1AK4.D.pssm"})
             >>> h5file = '1ak4.hdf5'
             >>>
             >>> #init the data assembler
-            >>> database = DataGenerator(chain1='C',
-            >>>                          chain2='D',
-            >>>                          pdb_source=pdb_source,
-            >>>                          pdb_native=pdb_native,
-            >>>                          pssm_source=pssm_source,
+            >>> database = DataGenerator([mutant],
             >>>                          data_augmentation=None,
             >>>                          compute_targets=['deeprank.targets.dockQ'],
```

Comment:
Do we need to edit dockQ.py?

Reply:
line 84(79) is actually an example situation. I'm not sure what dockQ does. We might need to give a different example, since we're not using deeprank for docking here.

This is the dockQ module.
https://github.com/DeepRank/deeprank/blob/master/deeprank/targets/dockQ.py

DockQ doesn't seem to do anything with the chain identifier inputs from deeprank, so I don't see what we should modify. We might need to design our own target module to incorporate the mutant severity values that we will use to train the network.

Reply:
Yes, I think I put my question wrongly. We would need to make our own module that does a similar check (what dockQ does for previous deeprank) 
dockQ computes structural similarity between complexes, a check basically. 

In our case, the targets are prediction of effects for a given mutation, so that would be- Damaging, benign or neutral. 
My question was are we are calling this attribute anywhere in the code? If so, then i need to change it.

Reply:
I don't fully understand what you mean by "this attribute". Do you mean dockQ's structural similarity or the prediction of effects for a mutation?
Target modules are used in the training phase of the neural network. So dockQ's output will be used if this module is included.

Reply:
Sorry about the confusion. 
By "this attribute" I mean dockQ. 

I noticed we are not calling it anywhere, so I merged the pull request. 
Also, designing a target module with mutant severity is next in my to-do list.

Reply:
actually, target modules are called in the preprocessing step but their output is used in the training phase. Sorry for misinforming you.

Reply:
Ok. 
So we need to build target module. I'll add this part in my slides.

Reply:
yes